infra/jitter.py

```diff
@@ -71,30 +78,34 @@ def make_plot(profile, upper_x_bound, output):
   outlier_y = []
 
   for idx, benchmark in enumerate(filtered):
-      benchmark_name = benchmark.get('benchmark', f'benchmark_{idx}')
```

Comment:
indentation was inconsistent, going with 2 spaces
in the future formatting would be nice